functionaltest/playwright/tests/chamtechE2E.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { test } from "@playwright/test";
+import { CookiesMethods } from "../components/methods/cookiesmethods.ui";
+import { DetailsMethods } from "../components/methods/detailsmethods.ui";
+import { HomeMethods } from "../components/methods/homemethods.ui";
+import { ProductMethods } from "../components/methods/productmethods.ui";
+
+test.only('Navigate to champion sports', async ({ page }) => {
```

Comment:
.only needs to remove

Reply:
Based on your suggentions comments updated